randovania/server/discord/preset_lookup.py

```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
 
 async def reply_for_layout_description(message: discord.Message, description: LayoutDescription):
     embed = discord.Embed(
-        title=f"Spoiler file for Randovania {description.randovania_version_text}",
+        title=f"Spoiler file (Generated with {description.randovania_version_text})",
     )
 
     if description.world_count == 1:
         preset = description.get_preset(0)
         embed.description = (
             f"{preset.game.long_name}, with preset {preset.name}.\n"
-            f"Seed Hash: {description.shareable_word_hash}\n"
-            f"Permalink: {description.permalink.as_base64_str}"
+            f"Seed Hash for {description.randovania_version_text}: {description.shareable_word_hash}"
```

Comment:
Description.randovania_version_text is wrong here, pretty sure that that's the randovania version that was used to generate the rdvgame. While the word hash here will always show from the current version.
You probably wanna use randovania.VERSION or something like that instead